test/l_percentage_transfer_manager.js

```diff
@@ -333,10 +348,26 @@ contract("PercentageTransferManager", accounts => {
            )
         });
 
+        it("Should successfully add the delegate", async() => {
```

Comment:
Please add a test before providing permission to make sure that unauthorized people are not allowed to use the function. Use `catchRevert` to assert revert.